one/src/migrations.rs

```diff
@@ -51,6 +51,11 @@ pub struct FromIpfsOpts {
     #[arg(long, env = "CERAMIC_ONE_LOCAL_NETWORK_ID")]
     local_network_id: Option<u32>,
 
+    /// Expect non sharded paths to blocks.
+    /// Sharded paths are organized into a two character prefix directories.
+    #[arg(long, env = "CERAMIC_ONE_NON_SHARDED_PATHS", default_value_t = false)]
+    non_sharded_paths: bool,
```

Comment:
FWIW, the sharding/key transformations are a configurable property of the blockstore and not a binary choice. But this should cover the "didn't actively configure sharding" case, and the most common option I guess.

https://github.com/ipfs/kubo/blob/master/docs/datastores.md